service/src/main/resources/db/migration/V509__nekku_product.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE public.nekku_product_meal_time AS ENUM ('BREAKFAST', 'LUNCH', 'SNACK', 'DINNER', 'SUPPER');
+
+CREATE TYPE public.nekku_product_meal_type AS ENUM ('VEGAN', 'VEGETABLE');
+
+CREATE TABLE nekku_product
+(
+    sku             text    PRIMARY KEY NOT NULL,
+    name            text    NOT NULL,
+    options_id      text,
```

Comment:
onko tämä viittaus toiseen tauluun vai mikä id?

Reply:
Joo, joka tapauksessa me tarkastellaan näitä, kun aloitetaan tekemään ruokatilausta. Pitäisikö tämä sitten jättää tekemättä?

Reply:
Ehkä jätetään foreign key constraint vielä pois, mutta indeksin voisi lisätä kuitenkin, tyyliin
`CREATE INDEX fk$nekku_product_options_id ON nekku_product(options_id);` tjsp